examples/sv3d/configs/sampling/sv3d_u.yaml

```diff
@@ -0,0 +1,124 @@
+model:
+  target: models.diffusion.VideoDiffusionEngine
```

Comment:
The name VideoDiffusionEngine seems confusing. Why not MultiViewDiffusionEngine?

Reply:
or simply DiffusionEngine

Reply:
Agree, I will put it as `MultiviewVideoDiffusionEngine` for better readability.